.gitpod.yml

```diff
@@ -5,8 +5,8 @@ ports:
     - port: 3000
       onOpen: open-preview
 tasks:
-    - command: >
-        learnpack run;
+    - command: 
```

Comment:
you cannot remove the `>` because that is needed if you command will take several lines, you can either do:

```
    - command: learnpack start;
```
or 
```
    - command: >
            learnpack start;
```
but you cannot do:

```
    - command:
            learnpack start;
```